variables.tf

```diff
@@ -638,6 +638,23 @@ variable "lua_ssl_cert" {
   default     = null
 }
 
+variable "kong_cluster_mtls" {
+  description = "Define what type of Cluster mTLS is required."
+  type        = string
+  default     = "shared"
+
+  validation {
+    condition     = contains(["pki", "shared"], var.kong_cluster_mtls)
+    error_message = "Invalid value - please choose pki or shared."
+  }
+}
```

Comment:
Should this validation reside under the module instead? Be good to state in the description the accepted values.

Reply:
Agree on the description, will update that - however all other variable validation takes place at the root module before it gets passed to the submodule whether that's ec2 or ecs, so best to keep it here